### DISCUSSION TITLE
Picture table and relations

### DISCUSSION THREAD

**Discussion by yaoyuandejuli** [opengisch/QField#3791] [Q&A]
Hello, 

In the tutorial  https://docs.qfield.org/how-to/pictures/ it says the following: 
_'You will need two tables. One table where the features are stored and one with a list of pictures.'_

Is this mandatory/necessary or is it just an exemple ? 

I am asking because i am working on a single table and about less than 5% of the time pictures don't get pushed.
I was wondering if it is related.

Kind regards ,

Hugo

**Comment by VxTedxV** [comment]
This is an example to be able to join multiple pictures to 1 feature, dynamically (no need to configure x fields with 1 picture each). If you know that you only want no more than 1, 2 or 3 pictures for each feature, you can simply configure 3 fields to record the path of each picture taken. The features with less than 3 pictures will just keep the other fields blank (null).
Your issue could be related to the weight of your pictures... (take too much time to upload ?)

**Reply by yaoyuandejuli** [reply]
Okay, thanks a lot for the answer that is exactly what I did . 

I limited the picture size so that they stay around 1 mb .
Most of the picture lost where taken by Iphone and since there is no way to currently access the folder directly on the phone they are stuck there.... 

I dont understand how qfield determine which file to upload when synchronizing or pushing.
Once the picture has been pushed once from the phone they will never be pushed again even if they are not on the cloud.
